lib/routes/chat.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+require 'sinatra'
+
+module Sinatra
+  module SlapiRoutes
+    # Sinatra Extension for Chat Access
+    # Its main functions are to:
+    #  1. Route Messages to Client
+    #     - Standard Text Chat
+    #     - Emote Chat
+    #     - Attachment Chat
+    module Chat
+      def self.registered(slapi)
+        # Handles a POST request for '/v1/attachment'
+        # @see https://api.slack.com/docs/message-attachments Slack Documentation
+        # @see https://api.slack.com/docs/messages/builder
+        #
+        # @param [Hash] params the parameters sent on the request
+        # @option params [String] :channel The Slack Channel ID (Public Channel Name works, private requires ID)
+        # @option params [Hash] attachments the information about the attachments
+        # @option attachments [String] :fallback
+        # @option attachments [String] :pretext
+        # @option attachments [String] :title
+        # @option attachments [String] :title_link
+        # @option attachments [String] :text
+        # @option attachments [String] :color ('#229954') defaults to @bot::GREEN(constant) if not specified
+        # @return [String] the resulting webpage
+        slapi.post '/v1/attachment' do
+          raise 'missing channel' unless params[:channel]
+          raise 'missing text' unless params[:attachments][:text]
+          raise 'missing fallback' unless params[:attachments][:fallback]
+          raise 'missing title' unless params[:attachments][:title]
+          channel = params[:channel]
+          slapi.chat_attachment(
+            {
+              pretext: params[:attachments][:pretext],
+              fallback: params[:attachments][:fallback],
+              title: params[:attachments][:title],
+              title_link: params[:attachments][:title_link],
+              text: params[:attachments][:text],
+              color: params[:attachments][:color] ? params[:attachments][:color] : '#229954'
+            },
+            channel
+          )
+          status 200
+          { 'message' => 'it worked' }.to_json
```

Comment:
As a separate issue we should address passing back the error status of these to the client making the call as defined here: https://api.slack.com/methods/chat.postMessage

Appears that argument errors are thrown: https://github.com/slack-ruby/slack-ruby-client/blob/684533af2e43561323e55f14ce6ce0eef7b3fadb/lib/slack/web/api/endpoints/chat.rb#L72

And likely status codes pass through on errors.